<launch>

    <!-- Static transform broadcaster -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="world_frame_broadcaster_node" args="0 0 0 0 0 0 1 world base_link" />

    <!-- Camea transform broadcaster -->
    <node pkg="reconstruction_3d_event_camera_pkg"  type="publish_camera_frame.py"  name="camera_frame_broadcaster_node" output="screen">
        <rosparam command="load" file="$(find reconstruction_3d_event_camera_pkg)/config/transforms_params.yaml"/>
    </node>
        
    <!-- Laser transform broadcaster -->
    <node pkg="reconstruction_3d_event_camera_pkg"  type="publish_laser_frame.py"  name="laser_frame_broadcaster" output="screen"/>

</launch>